Library/ViewModels/LoginToutViewModelTests.swift

```diff
@@ -139,7 +141,7 @@ final class LoginToutViewModelTests: TestCase {
     )
   }
 
-  func testFacebookLoginFlow_Success() {
+  func testFacebookLoginFlow_Success_WhenFBLoginDeprecationFlagDisabled() {
```

Comment:
can you write another test here that is "testFacebookLoginFlow_Success_WhenFBLoginDeprecationFlagEnabled"

Reply:
and below change the name of `testLogIntoEnvironment_SignInWithApple` to `testLogIntoEnvironment_SignInWithApple_WhenFBLoginDeprecationFlagDisabled` and also test its' opposite.